### PR TITLE
Inital Commit _signup

### PR DESCRIPTION
**Pull request by takuho24n**
This is "signup" branch for Sign up function. 
This is my first commitment of "signup"

### CODE REVIEW

**Review by aplazand20 — CHANGES_REQUESTED** [review]
There is this problem with the user show page. Please fix this first then push in this same branch.
image

**Comment by takuho24n** [comment]
> There is this problem with the user show page. Please fix this first then push in this same branch.
> image

Thank you for helping me! I've done with it!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def new
```
Comment (aplazand20) [inline app/controllers/users_controller.rb:2]: Change to Spaces: 2

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module UsersHelper
+    # Returns the Gravatar (http://gravatar.com/) for the given user.
+      def gravatar_for(user, options = { size: 50 })
+        gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
+        size = options[:size]
+        gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
+        image_tag(gravatar_url, alt: user.name, class: "gravatar")
+      end
+    end
```
Comment (aplazand20) [inline app/helpers/users_helper.rb]: Fix spaces here

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+
```
Comment (aplazand20) [inline app/models/user.rb:2]: Change to Spaces: 2

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<% provide(:title, 'Sign Up') %>
```
Comment (aplazand20) [inline app/views/users/new.html.erb:1]: Change all lines of code from Spaces: 2 to Spaces: 4

**Review by aplazand20 — CHANGES_REQUESTED** [review]
One last comment

**Review by aplazand20 — APPROVED** [review]
I approve this PR

**Pull request merged**